website/docs/r/storage_object_iam_policy.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: "google"
+page_title: "Google: google_storage_object_iam_policy"
+sidebar_current: "docs-google--storage-object-iam-policy"
+description: |-
+ Allows management of an IAM policy for a Google Storage Bucket Object.
+---
+
+# google\_storage\-_object\_iam\_policy
+
+Allows creation and management of an IAM policy for an existing Google Storage Bucket Object.
+
+## Example Usage
+
+```hcl
+resource "google_storage_bucket" "bucket" {
+  display_name = "%s"
+}
+
+resource "google_storage_object" "object" {
+  display_name = "%s"
+	bucket = "${google_storage_bucket.bucket.name}"
```

Comment:
Tabs/spaces problem. Generally we ask that documentation examples be run through `terraform fmt`.